res/controllers/engine-api.d.ts

```diff
@@ -61,6 +61,15 @@ declare namespace engine {
      * @param name Name of the control e.g. "play_indicator"
      * @returns Value of the control normalized to range of 0..1
      */
+    function getNormalizedValue(group: string, name: string): number;
+
+    /**
+     * Gets the control value normalized to a range of 0..1
+     * @param group Group of the control e.g. "[Channel1]"
+     * @param name Name of the control e.g. "play_indicator"
```

Comment:
It is better to use an slider as example
```suggestion
     * @param name Name of the control e.g. "rate"
```

Reply:
The same for the description above.